.gitignore

```diff
@@ -1,6 +1,7 @@
 *.
 .idea/
 target/
+cumulocity-sdk
 cumulocity-sdk/
```

Comment:
do we need both? Maybe can be replaced by regular expression

Reply:
The one with the slash did not match a symbolic link to sibling `cumulocity-sdk` directory, but supposedly matched a directory child directory with that name. Just to be safe, until we clean up the CI/CD scripts, I added it as a separate entry, because I'm not sure how and where on CI/CD that might be of any significance. Maybe overprotective.